index.html

```diff
@@ -32,16 +32,66 @@
   previewRender: markjax
 });
 ```
-
 And of course you can still write and use LaTeX:
 $$\sum_{n=1}^{\infty} 2^{-n} = 1$$
 </textarea>
 
     <script>
+    function insertAfterCursor(string) {      
+      var cursorCh = simplemde.codemirror.getCursor()["ch"]
+      var cursorLine = simplemde.codemirror.getCursor()["line"]
+      var anchor = {line: cursorLine, ch: cursorCh}
+
+      simplemde.codemirror.doc.replaceRange(string, anchor)
+    }
+
       var simplemde = new SimpleMDE({
-        previewRender: markjax,
+        previewRender: function(plainText, preview) {
+          setTimeout(function() {
+            markjax(plainText, preview);
+          }, 50);
+          return preview.innerHTML;
+        },
         spellChecker: false,
-        indentWithTabs: false
+        indentWithTabs: false,
+        toolbar: [
+          "bold",
+          "italic",
+          "heading",
+          "|",
+          {
+            name: "latexExpr",
+            action: function latexExpr(editor) {
+              insertAfterCursor("$ $")              
+            },
+            className: "fa fa-bar-chart-o",
+            title: "Latex Expression"
+          },
+          {
+            name: "latexEq",
+            action: function latexEq(editor) {
+              insertAfterCursor("$$ $$")
+            },
+            className: "fa fa-area-chart",
+            title: "Latex Equation"
+          },
+          "|",
+          "quote",
+          "unordered-list",
+          "ordered-list",
+          "|",
+          "link",
+          "image",
+          "|",
+          "preview",
+          "side-by-side",
+          "fullscreen",
+          "|",
+          "guide"
+        ],
+        "insertTexts": {
+          latexEq: ["", "%%"]
```

Comment:
I think this is not valid. We don't use `%` anywhere. https://github.com/NextStepWebs/simplemde-markdown-editor#configuration Check again what this configuration property means